packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte

```diff
@@ -441,17 +441,17 @@ function hasAnyConfiguration(provider: ProviderInfo) {
           </div>
         </div>
         <!-- providers columns -->
-        <div class="grow flex flex-wrap divide-gray-900 ml-2">
+        <div class="grow flex flex-wrap divide-gray-900 ml-2" role="region" aria-label="Provider Connections">
           <PreferencesConnectionsEmptyRendering
             message="{provider.emptyConnectionMarkdownDescription}"
             hidden="{provider.containerConnections.length > 0 || provider.kubernetesConnections.length > 0}" />
           {#each provider.containerConnections as container}
             {@const peerProperties = new PeerProperties()}
-            <div class="px-5 py-2 w-[240px]">
+            <div class="px-5 py-2 w-[240px]" role="region" aria-label="{container.name}">
               <div class="float-right">
-                <Tooltip tip="{provider.name} details" bottom>
+                <Tooltip tip="{provider.name} Details" bottom>
```

Comment:
AFAIK this change is not related to ARIA

Reply:
that is right. will fix.